app/models/propertyrepresentation/OrganisationLatestDetail.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.propertyrepresentation
+
+import java.time.LocalDateTime
+
+import play.api.libs.json.{Json, OFormat}
+
+case class OrganisationLatestDetail(
+                                     id: Long,
+                                     addressUnitId: Long,
+                                     organisationName: String,
+                                     organisationEmailAddress: String,
+                                     organisationTelephoneNumber: String,
+                                     representativeFlag: Boolean
+                                     //fixme add later effectiveFrom: LocalDateTime
```

Comment:
is this a deliberate FIXME?

Reply:
nope, removed

Reply:
there were other fixme's

Reply:
all addressed apart from 1 left deliberately